node/node_test.go

```diff
@@ -158,16 +161,19 @@ func setupFullNodes(t *testing.T, proto protocol.ConsensusVersion, verificationP
 		cfg, err := config.LoadConfigFromDisk(rootDirectory)
 		require.NoError(t, err)
 		cfg.Archival = true
-		_, err = data.LoadLedger(logging.Base().With("name", nodeID), ledgerFilenamePrefix, inMem, g.Proto, bootstrap, "", crypto.Digest{}, nil, cfg)
+		_, err = data.LoadLedger(logging.Base().With("name", nodeID), ledgerFilenamePrefix, inMem, g.Proto, bootstrap, g.ID(), g.Hash(), nil, cfg)
 		require.NoError(t, err)
 	}
 
 	for i := range nodes {
+		var nodeNeighbors []string
+		copy(nodeNeighbors, neighbors)
```

Comment:
`nodeNeighbors` has length 0 so nothing copied.
```golang
		copied := copy(nodeNeighbors, neighbors)
		fmt.Printf("copied = %d\n", copied)

copied = 0
```
so simple appending as below is fine